server/audit_math/ballot_polling.py

```diff
@@ -34,6 +35,11 @@ def get_sample_size(
         - A sample size dictionary containing sample sizes for different
           finishing probabilities
     """
+    # When a sampled ballot can't be found, count it as a vote for every loser
+    if sample_results:
+        for round_id, num_not_found in samples_not_found.items():
+            for loser in contest.losers:
+                sample_results[round_id][loser] += num_not_found
```

Comment:
Logic copied from the risk limit logic:

https://github.com/votingworks/arlo/blob/30eb4ee5ab68d18b70f730c0644952f6b2074bdd/server/audit_math/ballot_polling.py#L61-L64

Could ultimately place this somewhere shared but sufficient for now to unblock the Texas round 2 tomorrow